apps/mull-api/src/app/user/user.resolver.ts

```diff
@@ -48,15 +48,20 @@ export class UserResolver {
   ) {
     var user = await this.userService.getUser(userInput.id);
```

Comment:
This doesn't have to be a var
```suggestion
    const user = await this.userService.getUser(userInput.id);
```